drivers/scheduler/k8s/k8s.go

```diff
@@ -186,7 +184,7 @@ func (k *K8s) Init(schedOpts scheduler.InitOptions) error {
 		}
 	}
 
-	k.SpecFactory, err = spec.NewFactory(schedOpts.SpecDir, schedOpts.VolDriverName, k)
+	k.SpecFactory, err = spec.NewFactory(schedOpts.SpecDir, schedOpts.StorageProvisioner, k)
```

Comment:
This needs to be driver name, not provisioner. Driver name is pxd, aws, etc. Provisioner is whether it is in-tree or csi for that driver

Reply:
storageProvisioner param of NewFactory is only used for selecting appropriate templates. We need to chose either to use portworx spec with `kubernetes.io/portworx-volume` provisioner or `pxd.portworx.com` so this param helps to resolve it. If I pass pxd I can't make any conclusion on that or I can pass them both.

Reply:
Done